src/main/resources/com/dabsquared/gitlabjenkins/workflow/GitLabCommitStatusStep/config.groovy

```diff
@@ -0,0 +1,20 @@
+package com.dabsquared.gitlabjenkins.workflow.GitLabCommitStatusStep;
+
+f = namespace(lib.FormTagLib)
+
+f.entry(title:"Build name", field:"name") {
+    f.textbox()
+}
+
+f.entry(title:"Gitalb connection") {
```

Comment:
Typo `Gitalb` should be replaced with `Gitlab`

Reply:
Always GitLab if talking about the formal product name.